### ISSUE TITLE
How to run inference using 3mm model?

### ISSUE THREAD

**Issue by FrankWJW** [wasserth/TotalSegmentator#447]
Hi, 

My current project requires me to run both CT and MR dataset using TotalSegmentator as a baseline. The baseline should be the model train with full classes. To do that, I basically follow the same step in _train_nnunet.md_ with some changes:

In  _convert_dataset_to_nnunet.py_
I modified to adapt full classes instead of using part of classes:
```
from totalsegmentator.map_to_binary import class_map
# class_map = class_map_5_parts[class_map_name]
class_map = class_map['total']
```
And I adapt the following code in [https://github.com/wasserth/TotalSegmentator/issues/281#issue-2172097637](url) to resample the test images and labels to 3mm.

Then I downloaded the pretrained model:
`download_pretrained_weights(297)`
which corresponds to the pretrained model using total 117 classes on CT dataset.
Then running:
`nnUNetv2_predict -i path/to/imagesTs -o path/to/labelsTs_predicted -d 297 -c 3d_fullres -tr nnUNetTrainer_4000epochs_NoMirroring --disable_tta -f 0`

This gives warning message:
```
/usr/local/lib/python3.10/dist-packages/nnunetv2/inference/predict_from_raw_data.py:84: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  checkpoint = torch.load(join(model_training_output_dir, f'fold_{f}', checkpoint_name),
/usr/local/lib/python3.10/dist-packages/nnunetv2/utilities/plans_handling/plans_handler.py:37: UserWarning: Detected old nnU-Net plans format. Attempting to reconstruct network architecture parameters. If this fails, rerun nnUNetv2_plan_experiment for your dataset. If you use a custom architecture, please downgrade nnU-Net to the version you implemented this or update your implementation + plans.
  warnings.warn("Detected old nnU-Net plans format. Attempting to reconstruct network architecture "
```

To evaluate the data, I made a similar modification in the evaluate.py script to adapt the total classes:
`class_map = class_map['total']`

However, the final dice loss is far from the reported scores. As a comparison, I also run 1.5mm model as well but it works fine with little difference from the reported result.

I am using nnunetv2==2.4.1 and TotalSegmentator==2.7.0.

I wonder if there is an instruction to run inference with the pretrain model for task 297 (CT) and task 852 (MRI)? Thanks in advance!

**Comment by FrankWJW** [comment]
Now I found that evaluating the pretrain 3mm model using the default test set should give a good result. Here is the overall result I get - the dashed grey line is the averaged score from evaluate_results.txt and red is the result from 3mm model (task 297). May I know what is the actual result for the 3mm model? I don't think this has been reported in the paper as the dataset has been updated from 104 classes to 117.

![Image](https://github.com/user-attachments/assets/23dd9c77-b218-44df-9cb5-eafb3971771a)
